wca-guidelines.md

```diff
@@ -118,7 +118,6 @@ To be more informative, each Guideline is classified using one of the following
 
 - 7d+) [ADDITION] The temperature of the competition area should be 21 to 25 degrees Celsius.
 - 7f1a+) [CLARIFICATION] A full-size mat should have minimum dimensions of 30cm (left to right) by 25cm (front to back).
```

Comment:
Ideally we should have a more structured way to set expectations for the future, but for now I think we should put this here:
```suggestion
- 7f1a+) [CLARIFICATION] A full-size mat should have minimum dimensions of 30cm (left to right) by 25cm (front to back).
- 7f1a+) [EXPLANATION] O-rings (or equivalent modifications) may become required in the future. Organizers should attach them if possible, and contact the WRC if they expect permanent logistical barriers to doing so.
```

Reply:
I still don't like the idea of having an explanation for the future in the Guidelines. I think this would only make guidelines even more difficult to understand. I believe is okay to say it in the official announcement. Besides, this could become a WQAC in the future.

Note: You named the guideline with the same name as the other one

Reply:
Let's wait to see opinions from other WRC members though

Reply:
In general it's probably better to avoid adding more Regulations/Guidelines unless strictly needed, so I'm okay with leaving this out.